### DISCUSSION TITLE
apisix plugin limit-req . what about the delay ？

### DISCUSSION THREAD

**Discussion by michaelh1355** [apache/apisix#8204] [Q&A]
https://apisix.apache.org/zh/docs/apisix/plugins/limit-req/
https://apisix.apache.org/docs/apisix/plugins/limit-req/

根据文档If set to true, requests within the burst threshold would not be delayed.，请问delay是多少？能调节该值的大小吗？

**Comment by tokers** [comment]
No, the delay time is calculated according to the number in the queue.
